src/Form.js

```diff
@@ -41,10 +44,18 @@ useEffect (() => {
         setNews(news)
```

Comment:
I might be wrong but do you need to use the .then() when using fetch along with some try and catch error handling?